index.d.ts

```diff
@@ -24,626 +24,6 @@ declare global {
 
   ///////////////////////////////////////////////////////////////////////////////
   // Widget
-
-  interface WidgetAPI {
-    register(component: FunctionalWidget<any>): void
-    h(...args: any[]): FigmaDeclarativeNode
```

Comment:
was this line removed intentionally?

Reply:
Yes! As far as I can tell, there is no implementation for a 'h' hook in the widget API: https://github.com/figma/figma/blob/master/web/js/figma_app/plugin/jsvm_figma_api.ts